messaging/websockets-client/src/main/java/io/quarkus/ts/Pusher.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.ts;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.websocket.OnMessage;
+import javax.websocket.OnOpen;
+import javax.websocket.RemoteEndpoint;
+import javax.websocket.Session;
+import javax.websocket.server.ServerEndpoint;
+
+@ServerEndpoint("/push")
+@ApplicationScoped
+public class Pusher {
+
+    private Session session;
+
+    @OnOpen
+    public void onOpen(Session session) {
+        this.session = session;
+        //todo messages are not sent on opening of session. Need to investigate and create reproducer
+    }
+
+    @OnMessage
+    public void onMessage(String message) throws ExecutionException, InterruptedException, TimeoutException {
```

Comment:
the message is not used. Could you log the message (just for debugging purposes)